ranger/ext/img_display.py

```diff
@@ -298,7 +298,7 @@ def _generate_iterm2_input(self, path, max_cols, max_rows):
         content = self._encode_image_content(path)
         display_protocol = "\033"
         close_protocol = "\a"
-        if "screen" in os.environ['TERM']:
+        if any(x in os.environ['TERM'] for x in ["screen", "screen-256color", "tmux", "tmux-256color"]):
```

Comment:
Since we're using `in` we can just check for the "screen" and "tmux" substrings.

Reply:
Replace the `any(...)` with `os.environ["TERM"].startswith(("screen", "tmux"))`